code/modules/reagents/reagent_containers.dm

```diff
@@ -169,6 +169,14 @@
 		to_chat(user, "<span class='notice'>\The [src] is empty.</span>")
 		return 1
 
+	var/mob/living/list/eaters = list(
+		/mob/living/carbon,
+		/mob/living/simple_animal
+	)
+
+	if(!is_type_in_list(target, eaters))

```

Comment:
```suggestion
	if(!issynthetic(target) && !isipc(target))
```
IPC checks are handled elsewhere, hence why they're explicitly allowed.
The rest of the code except for the `return 0` can be removed with this. That should also probably be changed to `return FALSE`.

Reply:
Doesn't entirely work since /mob/living/bot returns FALSE for isSynthetic(), but I feel like that should honestly be changed.